config.py

```diff
@@ -32,6 +32,10 @@ class Config(object):
     INVITE_EMAIL_FROM = 'enquiries@digitalmarketplace.service.gov.uk'
     INVITE_EMAIL_SUBJECT = 'Your Digital Marketplace invitation'
 
+    CLARIFICATION_EMAIL_NAME = 'Digital Marketplace Admin'
+    CLARIFICATION_EMAIL_FROM = 'do-not-reply@digitalmarketplace.service.gov.uk'
```

Comment:
Cath wanted a `do-not-reply` email for the clarification questions. We aren't allowed to communicate with suppliers about their G7 questions directly, so makes sense they can't reply to our confirmation email.

But if they have problems with their account access (invites/passwords) then the enquiries email address is the one they should use to get Nicole to fix things up for them.